accelerator/api/apis.h

```diff
@@ -6,10 +6,10 @@
  * "LICENSE" at the root of this distribution.
  */
 
-#ifndef ACCELERATOR_APIS_H_
-#define ACCELERATOR_APIS_H_
+#ifndef API_APIS_H_
+#define API_APIS_H_
 
-#include "accelerator/common_core.h"
+#include "accelerator/api/core/core.h"
```

Comment:
It sounds weird to call it `api/core/core.h`.

Reply:
How about `accelerator/core/api.h`?

Reply:
`apis.h` is at higher level than `core.h`, since the functions in `apis.h` would call functions in `core.h` to process the API calls.
Therefore, if we use `core` as the second level directory name. then the files in `/core`, `/mam` would be moved out of the folders and be put at the same level as `apis.h` and `proxy_apis` doing.

Reply:
The tree structure would look like this
├── api
│ ├── apis.c
│ ├── apis.h
│ ├── core
│ ├── core.c
│ ├── core.h
│ ├── pow.c
│ ├── pow.h
│ ├── mam_core.c
│ ├── mam_core.h
│ ├── proxy_apis.c
│ ├── proxy_apis.h
│ ├── `/request`
│ ├── `/response`
│ └── `/serializer`

Reply:
Are `core.*, pow.* mam.* prorxy_api.*` under folder `core`?

Reply:
> Are `core.*, pow.* mam.* prorxy_api.*` under folder `core`?

It is "directory" core rather than "folder". Directory is of UNIX manner.

Reply:
So I will follow the structure I listed above to modify the directory